lib/signaling/v2/peerconnection.js

```diff
@@ -1268,9 +1268,14 @@ class PeerConnectionV2 extends StateMachine {
     if (!this._localDescription) {
       return null;
     }
+
+    // return the last revision that is set in the local description.
+    // not the one that we are in processing. they would be different if this function gets called for
+    // sending 'sync' message, while we were processing a new offer (but not yet sent the answer for it.)
```

Comment:
```suggestion
    // NOTE(mpatwardhan): If the most recent local description is an answer, and this method is called for
    // sending a "sync" message while the next remote offer is being processed, we need to send the most
    // recent stable description revision instead of the current description revision, which is supposed
    // to be for the next local answer.
```